components/resolved_ts/src/advance.rs

```diff
@@ -156,8 +157,7 @@ pub async fn region_resolved_ts_store(
     tikv_clients: Arc<Mutex<HashMap<u64, TikvClient>>>,
     min_ts: TimeStamp,
 ) -> Vec<u64> {
-    #[cfg(feature = "failpoint")]
-    (|| fail_point!("before_sync_replica_read_state", |_| regions))();
+    fail_point!("before_sync_replica_read_state", |_| regions.clone());
```

Comment:
Either the old implementation or the new implementation is wrong. /cc @NingLin-P